report_compassion/report/partner_communication.xml

```diff
@@ -227,7 +229,7 @@
             </div>
             <div id="letter">
                 <div id="address" class="right">
-                    <div id="pp">
+                    <div id="pp" t-attf-style="visibility: {{ 'visible' if display_pp else 'hidden' }};">
```

Comment:
Here instead of your t-attf-style attribute you can simply put `t-if="display_pp"`